Tzkt.Api/Websocket/Processors/OperationsProcessor.cs

```diff
@@ -184,6 +185,18 @@ void AddRange(HashSet<string> subs, IEnumerable<Operation> operations)
                     }
                 }
 
```

Comment:
This should be appended to the line `122`:
```cs
 || EntrypointsSubs.Count > 0
```
Otherwise you won't request txs from the DB if there are only entrypoints subs.